ompi/mpi/c/exscan.c

```diff
@@ -51,7 +51,14 @@ int MPI_Exscan(const void *sendbuf, void *recvbuf, int count,
 
     MEMCHECKER(
         memchecker_datatype(datatype);
-        memchecker_call(&opal_memchecker_base_isdefined, sendbuf, count, datatype);
+        if (0 != ompi_comm_rank(comm)) {
```

Comment:
We should check the send up and recvbuf on 0 as well. Just remove the 0 == rank and the test is correct.